src/hooks/baseSettings.tsx

```diff
@@ -11,14 +45,18 @@ export default function(useReducer: Function, useEffect: Function) {
     );
 
     const [state, setState] = reducerResult;
-    console.info("Reducer State", state);
+    console.info("Reducer State", state, settings);
 
     useEffect(() => {
       setState(settings);
     }, [settings]);
 
     useEffect(() => {
-      if (connectedResult && connectedResult.actionInfo) {
+      if (
+        connectedResult &&
+        connectedResult.actionInfo &&
+        settingsEvent === Settings.didReceiveSettings
```

Comment:
maybe its just my non-existing knowledge - is the other `didReceiveGlobalSettings` here not needed ?

Reply:
This "connected" event only gives us the instance settings. global have to be fetched separately

Reply:
ahh ok - nice! 

then dont mind me / this comment xD